packages/core/src/lib/toast/toast-container.svelte

```diff
@@ -0,0 +1,47 @@
+<!--
+@component
+
+For displaying toast messages.
+
+This is the "container" element that all toasts pushed using the
+`useToast` API will be added to. Add this component near the root of
+your application, like in the root layout component.
+
+It needs access to context from `provideToast`, which should also
+be added to the root layout component.
+-->
+<script lang="ts">
+import { fade, fly } from 'svelte/transition';
+import { flip } from 'svelte/animate';
+
+import { useToastState } from './context';
+import ToastBanner from './toast-banner.svelte';
+
+const { toasts, pageIsVisible } = useToastState();
+let visibilityState: DocumentVisibilityState;
+
+$: pageIsVisible.set(visibilityState === 'visible');
+</script>
+
+<svelte:document bind:visibilityState />
+
+<div
+  role="status"
+  aria-label="Toasts"
+  class="pointer-events-none fixed bottom-0 left-1/2 top-auto z-[9999] -translate-x-1/2 transform"
```

Comment:
We're missing an `overflow-hidden` on this element; in the storybook, a scrollbar appears while toasts are flying in if you're on desktop with a mouse (on macOS, it's likely worse on Windows)

[Nit] I think `z-[9999]` is equivalent to our `z-max`